internal/sbom/presets.go

```diff
@@ -1,5 +1,7 @@
 package sbom
 
+import "slices"
+
 import "github.com/containers/buildah/define"
```

Comment:
This looks... weird.

Reply:
Interestingly, even `gofumpt` doesn't care.

Fixed, thanks for noticing.